scripts/quests/otherAreas/Fishermans_Heart.lua

```diff
@@ -0,0 +1,147 @@
+-----------------------------------
+-- Fisherman's Heart
+-- Katsunaga !pos -4.726 -1.148 2.183
+-- Log ID: 4, Quest ID: 11
+-----------------------------------
+local quest = Quest:new(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.FISHERMAN_S_HEART)
+
+quest.reward =
+{
+    fame_area = xi.fameArea.WINDURST,
+    fame = 30,
+}
+
+quest.sections =
+{
+    -- Quest available, but fishing rank is 1 and skill < 20
+    -- (NOTE: Fishing Rank 0 is handled as default action)
+    {
+        check = function(player, status, vars)
+            return
```

Comment:
Formatting-wise, return <first cond> <and/or> newline, then following ones with 4-space indent